packages/connect-popup/README.md

```diff
@@ -1,15 +1,19 @@
 # @trezor/connect-popup
 
-Build `@trezor/connect` popup hosted on `connect.trezor.io/*/popup.html`
+Build `@trezor/connect` popup, which serves as a UI for communication with Trezor device via [@trezor/connect](../connect). UI shows a loading screen until a request from [@trezor/connect-iframe] is received.
```

Comment:
`@trezor/connect-web`

UI shows a loading screen until communication channel between popup and [@trezor/connect-iframe] is initialized